04_environment/index.tex

```diff
@@ -1 +1,400 @@
+\section{Common Pool}
+\subsection{Common Pool Background}
+
+We are designing a game where 6-islands (agents) are having access to a common-pool resource. In micro-level, each island intends to maximise its utility while in macro-level, all the islands want to maintain sustainability and be protected by the upcoming disaster. The above description specifies a collective action problem where n-agents (6 in our case) are seeking access to a common pool resource that is sufficient to satisfy some agents but not to satisfy all of them.\\
+
+We have implemented a Linear Public Goods (LPG) Game where all the agents individually own some resources and try to maintain and increase them, by foraging, but also contribute a part of them to the common pool. This common pool is used for the payment of expenses of the institutional roles and for the protection towards the disaster.\\
+
+In more detail, the common pool is represented by a structure with following characteristics:
+
+\begin{itemize}
+    \item Common Pool Threshold
+    \item Amount of Resources 
+\end{itemize}
+
+According to the specifications of an LPG Game, the agents should be able to perform the following actions:  
+
+\begin{itemize}
+    \item Request Resource from the Common Pool 
+    \item Contribute Resource to the Common Pool 
+\end{itemize}
+
+And the Common Pool is responsible for two more actions: 
+
+\begin{itemize}
+    \item Mitigate Disaster
+    \item Deplete islands after a disaster
+\end{itemize}
+
+\newpage
+\subsection{functions within the Common pool}
+
+For each of these action a relevant function has been implemented as follows:\\
+
+\begin{table}[h]
+\begin{center}
+\begin{tabular}
+{|p{1.1in}|p{1.1in}|p{1.1in}|p{1.1in}|} \hline
+\textbf{Name} & \textbf{Description} & \textbf{Input} & \textbf{Outcome} \\ \hline
+\texttt{islandDistribute} & Distribute Resources from Common Pool to Island upon Request & -Requested Amount of Resources\newline -Island Name & Resources subtracted from CP and added to island \\ \hline
+\texttt{islandContribute} & Contribute Resources from Island to Common Pool & -Donated Amount of Resources\newline -Island Name & Resources subtracted from Island and added to CP \\ \hline
+\texttt{DisasterMitigate} & Mitigate the Disaster with the Amount of Resources in the Common Pool & - Effect of Disaster to each island\newline - Proportional effect of Disaster to each island (magnitude of disaster divided by the distance of the island from epicenter) & Resources in CP were exploited for the mitigation of the disaster and remaining effect has to be mitigated by each island \\ \hline
+\texttt{islandDeplete} & Deplete Islands after the disaster (with the amount of disaster that couldn't be mitigated by CP) & -Remaining effect for each Island\newline -List of Islands & The remaining proportional effect of the disaster to each island is causing the depletion of resources \\ \hline
+\end{tabular}
+\caption{\label{tab:table-name}Common Pool's main functions}
```

Comment:
why did you put label into the caption??
suggestion: put it AFTER the caption

Reply:
after or before the caption doesn't matter but definitely not inside..

Reply:
and the caption + label should go before the end of the table